demisto_sdk/commands/common/schemas/integration.yml

```diff
@@ -56,6 +56,9 @@ mapping:
     type: str
   autoconfiginstance:
     type: bool
+  supportlevelheader:
+    type: str
+    enum: ['xsoar', 'partner', 'community']
```

Comment:
will this break if the header isnt present?

Reply:
no, because the **supportlevelheader** is **optional**.